.github/workflows/release.yml

```diff
@@ -29,127 +29,37 @@ jobs:
-        run: echo "::set-env name=NIGHTLY_FLAG::--nightly"
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v1
-        with:
-          python-version: ${{ matrix.python-version }}
-      - name: Build macOS wheels
-        if: github.event_name == 'schedule' || github.event_name == 'release' || matrix.python-version == '3.5'
-        run: |
-          python3 --version
-          python3 -m pip install delocate wheel setuptools
-
-          bash tools/ci_testing/install_bazel_macos.sh $BAZEL_VERSION
-          bash tools/ci_testing/addons_cpu.sh
-
-          bazel build \
-            -c opt \
-            --copt -mmacosx-version-min=10.13 \
-            --linkopt -mmacosx-version-min=10.13 \
-            --noshow_progress \
-            --noshow_loading_progress \
-            --verbose_failures \
-            --test_output=errors \
-            build_pip_pkg
-
-          bazel-bin/build_pip_pkg artifacts $NIGHTLY_FLAG
-          for f in artifacts/*.whl; do
-            delocate-wheel -w wheelhouse $f
-          done
-      - uses: actions/upload-artifact@v1
-        if: github.event_name == 'schedule' || github.event_name == 'release' || matrix.python-version == '3.5'
-        with:
-          name: ${{ runner.os }}-wheels
-          path: wheelhouse
-
-  windows-release-wheel:
-    name: Build release wheels for Windows
-    runs-on: windows-latest
-    strategy:
-      matrix:
-        python-version: ['3.5', '3.6', '3.7']
-    steps:
-      - if: github.event_name == 'schedule'
-        run: echo "::set-env name=NIGHTLY_FLAG::--nightly"
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v1
-        with:
-          python-version: ${{ matrix.python-version }}
-      - name: Build Windows wheels
-        env:
-          TF_NEED_CUDA: 0
-          BAZEL_VC: "C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/"
+          OS: ${{ runner.os }}
         shell: bash
-        if: github.event_name == 'schedule' || github.event_name == 'release' || matrix.python-version == '3.5'
-        run: |
-          python -m pip install wheel setuptools
-          curl -sSOL https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-windows-x86_64.exe
-          export BAZEL_PATH=/d/a/addons/addons/bazel-${BAZEL_VERSION}-windows-x86_64.exe
-          bash ./tools/ci_testing/addons_cpu.sh
-
-          ./bazel-${BAZEL_VERSION}-windows-x86_64.exe build \
-            -c opt \
-            --enable_runfiles \
-            --noshow_progress \
-            --noshow_loading_progress \
-            --verbose_failures \
-            --test_output=errors \
-            build_pip_pkg
-          bazel-bin/build_pip_pkg artifacts $NIGHTLY_FLAG
+        run: bash .github/workflows/make_wheel_${OS}.sh
       - uses: actions/upload-artifact@v1
         if: github.event_name == 'schedule' || github.event_name == 'release' || matrix.python-version == '3.5'
```

Comment:
Can we remove this check? I believe the upload will have no affect if there are no whl's created in the step prior?

Reply:
We can always try.

Reply:
I doesn't seem to work. I'll revert the commit.